tests/functional/test_task_submit.py

```diff
@@ -45,7 +45,7 @@ def test_exlude_recursive(run_line, load_api_fixtures, go_ep1_id, go_ep2_id):
     # batch
     batch_input = "abc /def\n"
     result = run_line(
-        "globus transfer --exclude *.txt --batch "
+        "globus transfer --exclude *.txt --batch - "
```

Comment:
Although it's testing a click behavior, in some sense, I think we should also add a variant of this test which writes `abc /def\n` to a temp file and runs `--batch` on that file.
That way, we have an explicit test for the new behavior.